sql/sql_show.cc

```diff
@@ -9756,6 +9804,8 @@ ST_SCHEMA_TABLE schema_tables[]=
    fill_schema_applicable_roles, 0, 0, -1, -1, 0, 0},
   {"CHARACTER_SETS", charsets_fields_info, 0,
    fill_schema_charsets, make_character_sets_old_format, 0, -1, -1, 0, 0},
+  {"CHECK_CONSTRAINTS", check_constraints_fields_info, 0,
+   get_all_tables, 0, get_check_constraints_record, 1, 2, 0, OPTIMIZE_I_S_TABLE|OPEN_TABLE_ONLY},
```

Comment:
idx_field1 and idx_field2 are ok to be 1 and 2 if the order of the fields is CONSTRAINT_SCHEMA, TABLE_NAME.

Reply:
Also, respect 80 character rule please.